test/e2e/throughputanomalydetection_test.go

```diff
@@ -101,7 +101,7 @@ func prepareFlowTable(t *testing.T, connect *sql.DB) {
 // Example output: Successfully created Throughput Anomaly Detection job with name tad-eec9d1be-7204-4d50-8f57-d9c8757a2668
 func testThroughputAnomalyDetectionAlgo(t *testing.T, data *TestData, connect *sql.DB) {
 	prepareFlowTable(t, connect)
-	stdout, jobName, err := tadrunJob(t, data, "ARIMA")
+	stdout, jobName, err := tadrunJob(t, data, "ARIMA", "e2e")
```

Comment:
If you just want to skip the agg_type check for these tests, maybe define a constant string like SkipAggType or AggTypeNone and use it here? e2e is a little bit confusing.